scripts/provisioning/s3.init.conf.tmpl

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Copyright

Reply:
I realized it later that we cannot add copyright to this file as this file is a json file, (confstore would load it as json file) and we cannot add comments to json file.

Reply:
Added (after changing the format from json to yaml)